gst/tensor_reposrc/tensor_reposrc.h

```diff
@@ -0,0 +1,81 @@
+/**
+ * GStreamer
+ * Copyright (C) 2005 Thomas Vander Stichele <thomas@apestaart.org>
+ * Copyright (C) 2005 Ronald S. Bultje <rbultje@ronald.bitfreak.net>
+ * Copyright (C) 2018 Samsung Electronics Co., Ltd.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ */
+
+/**
+ * @file	tensor_reposrc.h
+ * @date	19 Nov 2018
+ * @brief	GStreamer plugin to handle tensor repository
+ * @see		https://github.com/nnsuite/nnstreamer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ */
+
+#ifndef __GST_TENSOR_REPOSRC_H_
+#define __GST_TENSOR_REPOSRC_H__
+
+#include <gst/gst.h>
+#include <gst/base/gstpushsrc.h>
+#include <tensor_repo.h>
+
+G_BEGIN_DECLS
+#define GST_TYPE_TENSOR_REPOSRC \
+  (gst_tensor_reposrc_get_type())
+#define GST_TENSOR_REPOSRC(obj) \
+  (G_TYPE_CHECK_INSTANCE_CAST((obj),GST_TYPE_TENSOR_REPOSRC,GstTensorRepoSrc))
+#define GST_TENSOR_REPOSRC_CLASS(klass) \
+  (G_TYPE_CHECK_CLASS_CAST((klass),GST_TYPE_TENSOR_REPOSRC,GstTensorRepoSrcClass))
+#define GST_IS_TENSOR_REPOSRC(obj) \
+  (G_TYPE_CHECK_INSTANCE_TYPE((obj),GST_TYPE_TENSOR_REPOSRC))
+#define GST_IS_TENSOR_REPOSRC_CLASS(klass) \
+  (G_TYPE_CHECK_CLASS_TYPE((klass),GST_TYPE_TENSOR_REPOSRC))
+typedef struct _GstTensorRepoSrc GstTensorRepoSrc;
+typedef struct _GstTensorRepoSrcClass GstTensorRepoSrcClass;
+
+/**
+ * @brief GstTensorRepoSrc data structure.
+ *
+ * GstTensorRepoSrc inherits GstBaseSink.
+ */
+struct _GstTensorRepoSrc
+{
+  GstPushSrc parent;
+  GstBaseSink element;
```

Comment:
GstBaseSink element;
this line is unnecessary.

Reply:
Right. I forgot to delete. Thanks.